### ISSUE TITLE
Late 2018 refreshment

### ISSUE THREAD

**Issue by mischah** [h5bp/html5please#409]
html5please is pretty much unmaintained and needs a major update. 

## The idea

- review and update each and every feature
- collect and add new features

I hope we find people willing to contribute to make that happen.

### New features

If you find a new feature that isn’t part of html5please yet please feel free comment on this issue and add a new issue before starting to work on it.

### Update existing features

Feel free to pick one of the following to start working on it:

1. Eventsource
1. animations
1. audio
1. background-image
1. border-image
1. border-radius
1. box-reflection
1. box-shadow
1. box-sizing
1. calc
1. canvas
1. classlist
1. contenteditable
1. cors
1. css-filters
1. css-fixed
1. css-sticky
1. css3-colors
1. datalist
1. dataset
1. details
1. dialog
1. dom
1. dragndrop
1. es5
1. es6
1. exclusions
1. fetch-api
1. fileapi
1. filesystem
1. flexbox
1. font-feature-settings
1. fontface
1. form-validation
1. formdata
1. fullscreen
1. geolocation
1. gradients
1. grids
1. history
1. html5semantic
1. hyphens
1. iframe-sandbox
1. iframe-seamless
1. iframe-srcdoc
1. img-srcset
1. indexeddb
1. input-color
1. input-datetime
1. input-number
1. input-placeholder
1. input-range
1. input-search
1. interaction-media-features
1. json
1. localstorage
1. masks
1. matchmedia
1. mediaqueries
1. menu
1. meter
1. microdata-domapi
1. minmaxwh
1. multibackgrounds
1. multicolumn
1. offline
1. ol-reversed
1. opacity
1. paged-media
1. picture
1. pointer-events
1. postmessage
1. progress
1. pseudo-elements
1. regions
1. rem
1. requestanimationframe
1. ruby
1. scoped-css
1. selectors
1. supports
1. svg
1. text-overflow
1. text-shadow
1. text-stroke
1. track
1. transforms
1. transitions
1. vibration
1. video
1. viewport-units
1. webcomponents
1. webgl
1. webp
1. websockets
1. websql
1. webworker

**Comment by mischah** [comment]
Hej @h5bp/html5please, @paulirish,

would you please consider spreading the word, about this?

Because:

- I would like to get reviews to the PRs I’ll create 
- I hope that this will 
- I think this refreshment should be accomplished pretty soon.
  - Otherwise I’m afraid that htmlplease will be pretty worthless soon
- I hope that we will engage people to contribute on a frequent basis

**Comment by paulirish** [comment]
@mischah 	i just gave a quick review now, but I'll likely be unable to review again. You should feel free to be the primary reviewer now. :)

Thanks very much for spending time on this and whipping it back into shape!

**Comment by mischah** [comment]
Thanks Paul 😊

**Comment by mischah** [comment]
Hej @coliff, just in case you would like to participate on this issue:

Reference this isuue in your PR to track the checkboxes above. Just in case you aren’t able to check them by yourself.

Cheers, Michael

**Comment by yukulele** [comment]
shouldn't [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) also be added?
